src/components/Modal/BaseModal.tsx

```diff
@@ -176,6 +176,7 @@ function BaseModal(
 
     return (
         <ReactNativeModal
+            onClick={(e) => e.stopPropagation()}
```

Comment:
I decided to fix it globally

Reply:
Comment added

Reply:
I always try to avoid putting an issue reference on a comment, except a complex one, as I think it indicates that the comment doesn't explain what the code does well. I think the current comment is easy to understand, but if it's not, let me know what word should be updated.